src/core/Akka.MultiNodeTestRunner/Program.cs

```diff
@@ -64,13 +71,40 @@ class Program
         ///                  as the test binary.
         ///     </description>
         /// </item>
+        /// <item>
+        ///     <term>-Dmultinode.listen-address={ip}</term>
+        ///     <description>
+        ///             Determines the address that this multi-node test runner will use to listen for log messages from
+        ///             individual NodeTestRunner.exe processes.
+        /// 
+        ///             Defaults to 127.0.0.1
+        ///     </description>
+        /// </item>
+        /// <item>
+        ///     <term>-Dmultinode.listen-port={port}</term>
+        ///     <description>
+        ///             Determines the port number that this multi-node test runner will use to listen for log messages from
+        ///             individual NodeTestRunner.exe processes.
+        /// 
+        ///             Defaults to 6577
+        ///     </description>
+        /// </item>
         /// </list>
         /// </summary>
         static void Main(string[] args)
         {
+            OutputDirectory = CommandLine.GetProperty("multinode.output-directory");
```

Comment:
This would return null if this property doesn't exist and therefore Path.Combine would throw an ArgumentNullException later on. Should probably return an empty string instead of null.